Error handling is an important part of programming. It helps to manage and deal with unexpected conditions that arise while a program is running. 

PHP provides several ways to handle errors:

1. **Simple "die()" statements:** This function prints a message and exits the current script.

```php
<?php
   $file=fopen("welcome.txt","r");
   if(!$file) {
      die("Error: Unable to open the file");
   }
?>
```

2. **Custom error handler:** You can define your own function to handle the error. The `set_error_handler()` function tells PHP to use your custom error handler.

```php
<?php
    function customError($errno, $errstr) {
        echo "<b>Error:</b> [$errno] $errstr";
    }

    //set error handler
    set_error_handler("customError");

    //trigger an error
    echo($test);
?>
```

In this code, `customError()` is a user-defined function that formats the error message. `$errno` holds the level of the error and `$errstr` holds the error message. The `set_error_handler()` function tells PHP to use this function when an error occurs.

3. **Error Reporting:** PHP has various levels of error reporting for different kinds of error. The `error_reporting()` function sets the error_reporting directive at runtime.

```php
<?php
    //Turn off all error reporting
    error_reporting(0);

    //Report simple running errors
    error_reporting(E_ERROR | E_WARNING | E_PARSE);

    //Reporting E_NOTICE can be good to (to report uninitialized variables or catch variable name misspellings ...)
    error_reporting(E_ERROR | E_WARNING | E_PARSE | E_NOTICE);

    //Report all errors except E_NOTICE
    //This is the default value set in php.ini
    error_reporting(E_ALL ^ E_NOTICE);

    //Report all PHP errors
    error_reporting(E_ALL);
?>
```

4. **Exceptions:** PHP 5 has an exception model similar to that of other programming languages. Exceptions are used for handling errors and other exceptional events.

```php
<?php
    function checkNum($number) {
        if($number>1) {
            throw new Exception("Value must be 1 or below");
        }
        return true;
    }

    try {
        checkNum(2);
        echo 'If you see this, the number is 1 or below';
    }

    catch(Exception $e) {
        echo 'Message: ' .$e->getMessage();
    }
?>
```

In this code, the `checkNum()` function checks if a number is greater than 1. If it is, the function throws an exception. In the `try` block, if the `checkNum()` function throws an exception, the `catch` block catches it and displays the error message.